integreat_cms/cms/views/pages/page_actions.py

```diff
@@ -827,3 +828,52 @@ def render_mirrored_page_field(request, region_slug, language_slug):
             "page_form": page_form,
         },
     )
+
+
+@require_POST
+@permission_required("cms.change_page")
+# pylint: disable=unused-argument
+def refresh_date(
+    request,
+    page_id,
+    region_slug,
+    language_slug,
+):
+    """
+    Refresh the date for all translations of a corresponding page
+
+    :param request: The current request
+    :type request: ~django.http.HttpResponse
+
+    :param page_id: The id of the page of the current page form
+    :type page_id: int
+
+    :param region_slug: The slug of the current region
+    :type region_slug: str
+
+    :param language_slug: The slug of the current language
+    :type language_slug: str
+
+    :return: A redirection to the :class:`~integreat_cms.cms.views.pages.page_form_view.PageFormView`
+    :rtype: ~django.http.HttpResponseRedirect
+    """
+    region = request.region
+    page = get_object_or_404(region.pages, id=page_id)
+
+    # Consider only the last version of each translation
+    page_translations = page.translations.distinct("page__pk", "language__pk")
```

Comment:
```suggestion
    page_translations = page.translations.filter(
        language__in=region.active_languages
    ).distinct("page__pk", "language__pk")
```
We could also filter the translations to return only active ones.